python/packages/autogen-ext/src/autogen_ext/code_executors/__init__.py

```diff
@@ -1,4 +1,4 @@
-from ._azure_container_code_executor import ACADynamicSessionsCodeExecutor, TokenProvider
-from ._docker_code_executor import DockerCommandLineCodeExecutor
+from .azure import ACADynamicSessionsCodeExecutor, TokenProvider
+from .docker import DockerCommandLineCodeExecutor
 
 __all__ = ["DockerCommandLineCodeExecutor", "TokenProvider", "ACADynamicSessionsCodeExecutor"]
```

Comment:
Do you want to keep this or force imports from the respective extra namespaces?

Reply:
To prevent problems with optional dependencies I think we need to force the import from the nested module

Reply:
Revmoed then